hardhat.config.js

```diff
@@ -0,0 +1,14 @@
+require("@nomiclabs/hardhat-waffle");
+
+const ALCHEMY_API_KEY = "VF51FI3Duv3wY2OLO8_RuTIvxoIsNPMh";
+const RINKEBY_PRIVATE_KEY = "1e953f2feee97923de0badccf7c514bc7b570d4bc90732789328a0c7ddb622f9";
```

Comment:
We should create a .env file and move these keys to this file. We need to keep our private key secure.